### PR TITLE
docker: clarify environment config options

### PR DESCRIPTION
**Pull request by gustavosbarreto**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,6 +1,28 @@
+# PLEASE DO NOT EDIT THIS FILE; modify '.env.override' file instead
+# This is in order to avoid conflict with upstream code when updating to a newer version
+
+# Automatically increased once a new release is out
 SHELLHUB_VERSION=v0.3.4
```
Comment (otavio) [inline .env:5]: This seems outdated.

File: .env [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+# NOTICE: When enabling automatic HTTPS are recommended to set `SHELLHUB_HTTP_PORT=0` to avoid exposing plain HTTP service to the internet
+SHELLHUB_AUTO_SSL=false
+
+# Domain of the server (only required if automatic HTTPS is enabled)
+# Values: a valid domain name
+SHELLHUB_DOMAIN=localhost
+
+# Internal to our hosting service. - don't worry about it
 SHELLHUB_HOSTED=false
 SHELLHUB_AUTO_SSL=false
```
Comment (otavio) [inline .env:28]: Missing newline at end of file.

File: .env (hunk 1 above)
Comment (gustavosbarreto) [reply .env:5]: The last release that we made from master branch was 0.3.4.

Comment (otavio) [reply .env:5]: You're right :)

**Pull request merged**